apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.html

```diff
@@ -186,7 +185,7 @@ <h1>Sorry, No Data Found!</h1>
                 </div>
 
                 <div class="account clearfix" *ngIf="plData && plData?.incomeStatment"
-                    [ngStyle]="{'border-bottom':plData?.incomeStatment.incomeBeforeTaxes.type === 'DEBIT' ? '1px solid #dbdde3' : 'none' }">
+                    [ngStyle]="{'border-bottom':plData?.incomeStatment.incomeBeforeTaxes.type === 'DEBIT' ? 'none' : 'none' }">
```

Comment:
NgStyle is no longer needed as condition failure or success are doing the same thing

Reply:
Done